phi/tools/telegram.py

```diff
@@ -0,0 +1,35 @@
+from typing import Union
+import httpx
+from phi.tools import Toolkit
+
+
+class TelegramTools(Toolkit):
+    base_url = "https://api.telegram.org"
+
+    def __init__(self, token: str, chat_id: Union[str, int]):
+        super().__init__(name="telegram")
+        self.token = token
+        self.chat_id = chat_id
+
+        if not self.token or not self.chat_id:
+            raise ValueError("Token or chat ID not provided.")
+
+        self.register(self.send_message)
```

Comment:
```suggestion
    def __init__(self, chat_id: Union[str, int], token: Optional[str] = None):
        super().__init__(name="telegram")

        self.token = token or getenv("TELEGRAM_TOKEN")
        if not self.token:
            logger.error("TELEGRAM_TOKEN not set. Please set the TELEGRAM_TOKEN environment variable.")

        self.chat_id = chat_id

        self.register(self.send_message)
```

Reply:
Thank you for the suggestion! I’ve reviewed it and implemented the changes accordingly 🚀 🚀